src/pages/our_ambassadors.astro

```diff
@@ -545,6 +545,20 @@ import AmbassadorCard from "@components/AmbassadorCard.astro";
       >
         Sofia Stamouli...
       </AmbassadorCard>
+
+      <AmbassadorCard
+        name="Zohaib Anwar"
+        img="zohaibanwar.jpeg"
+        country="ca"
+        github="anwarMZ"
+        linkedin="mzohaibanwar"
+        twitter="zohaibanwar_"
+      >
+        Zohaib is a Senior Postdoctoral Researcher at the
```

Comment:
I'd recommend shortening the mini bio a bit so that the icons at the bottom aren't pushed too much down the block (check [the preview ](https://deploy-preview-224--nextflow-staging.netlify.app/our_ambassadors.html)to see how it looks now).